tutorial.rst

```diff
@@ -8,467 +8,11 @@ Tutorial
-    - for distributed dipoles, the nth line correspond to the amplitude of the nth vertex in the source mesh.
-
-Example for isolated dipoles:
-
-.. image:: _static/dipActiv.png
-   :width: 600 px
-   :alt: Dipole positions
-   :align: center
-
-For distributed sources, a source mesh describes their support. This is a detailed
-mesh generally covering the whole cortex. The mesh size should not exceed 35 000 points.
-The source amplitude is represented as continuous, and linear on each of the mesh triangles.
-The source orientation is modeled as piecewise constant, normal to each of the mesh triangles.
-
-.. image:: _static/cortex.png
-   :width: 300 px
-   :alt: Cortex
-   :align: center
-
-Isolated sources are the superposition of current dipoles, each of which is defined by its position and its moment.
-
-.. _sec.sensors:
-
-Sensors
--------
-
-For EEG, the sensors are defined by the list of the x-y-z coordinates of the electrode
-positions. The electrodes are considered punctual and are called *patches*.
-The MEG sensor description is more complex:
-The MEG sensor definition is provided in a text file, in which each line provides the position of the sensor, and additional information such as its orientation or its name.
-
-Sensors may have names (labels) in the first column of the file (it has to contains at least one character to be considered as label).
-
-More precisely, *omiting the first column which can contain a label* there are 4 options for defining EEG, EIT or MEG sensors:
-
-    - 1 line per sensor and 3 columns (typically for EEG sensors or MEG sensors without orientation or EIT punctual patches) :
-         - the 1st, 2nd and 3rd columns are respectively position coordinates x, y, z of sensor
-    - 1 line per sensor and 4 columns (spatially extended EIT sensors (circular patches) :
-         - the 1st, 2nd and 3rd columns are respectively position coordinates x, y, z of sensor
-         - the 4th column is the patche radius (unit relative to the mesh)
-    - 1 line per sensor and 6 columns (typically for MEG sensors) :
-         - the 1st, 2nd and 3rd are respectively position coordinates x, y, z of sensor
-         - the 4th, 5th and 6th are coordinates of vector orientation
-    - 1 line per integration point for each sensor and 8 columns (typically for MEG realistic sensors with coils, or gradiometers) :
-         - the 1st column is sensors names
-         - the 2nd, 3rd and 4th are respectively position coordinates x, y, z of sensor
-         - the 5th, 6th and 7th are coordinates of vector orientation
-         - the 8th is the weight to apply for numerical integration (related to sensor name)
-
-An example of MEG sensor description:
-
-.. image:: _static/sensors-grad.png
-   :width: 600 px
-   :alt: Sensor description
-   :align: center
+.. toctree::
+   :maxdepth: 2
+   :numbered:
+
+   Data.rst
+   Commands.rst
+   Examples.rst
+   Appendix.rst
```

Comment:
just use:

```
.. contents:: Table of Contents
   :local:
   :depth: 2

.. include:: Data.rst
.. include:: Commands.rst
.. include:: Examples.rst
.. include:: Appendix.rst
```
and you have your include and you can split the files in subfiles. I tried it works.